static/js/browser2/browser_page.tsx

```diff
@@ -20,17 +20,33 @@
 
 import React from "react";
 import { ArcSection } from "./arc_section";
-
+import { ObservationChartSection } from "./observation_chart_section";
 interface BrowserPagePropType {
   dcid: string;
+  isPlaceStatVarNode: boolean;
   nodeName: string;
+  statVarId: string;
 }
 
 export class BrowserPage extends React.Component<BrowserPagePropType> {
   render(): JSX.Element {
+    const pageName = this.props.isPlaceStatVarNode
+      ? this.props.statVarId + " in " + this.props.nodeName
```

Comment:
the js equivalent format string is specified with backticks:
`${this.props.statVarId } in ${this.props.nodeName}`

Reply:
done